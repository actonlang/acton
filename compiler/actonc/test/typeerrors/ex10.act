def f(x):
    try:
        y = x+2
    except int:
        pass
    
    
