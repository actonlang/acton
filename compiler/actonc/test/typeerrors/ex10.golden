Building file test/typeerrors/ex10.act
  Compiling ex10.act for development
[error]: Constraint violation
     +--> ex10@4:5-4:15
     |
   4 |     except int:
     :     ^--------- 
     :     `- __builtin__.int must be a subclass of __builtin__.Exception
-----+
